rfcs/proposed/range_algorithms_3/README.md

```diff
@@ -0,0 +1,48 @@
+# Support the third portion of the oneDPL Range APIs
+
+## Introduction
+Based on statistics (observing C++ code within github.com) for the usage of popular algorithms, the following
+range-based APIs are suggested to be implemented next in oneDPL.
+`swap_ranges`, `reverse`, `reverse_copy`, `unique`, `unique_copy`, `set_intersection`, `set_union`, `set_difference`, `set_symmetric_difference`,
+`includes`, `nth_element`, `partition`, `partition_copy`, `remove_copy`, `remove_copy_if`, `lexicographical_compare`, `rotate`, `rotate_copy`,
+`uninitialized_copy`, `uninitialized_move`, `uninitialized_fill`, `uninitialized_default_construct`, `uninitialized_value_construct`, `destroy`
+
+## Motivations
+The feature is proposed as the next step of range-based API support for oneDPL.
+
+### Key Requirements
```

Comment:
Let's add the feature testing macro update to the requirements, so that it is not forgotten again.

Reply:
It's been added, see 5d3ad9b078072838b2f110a056f5c1727b7ba7d1